pkgs/by-name/mu/muse-sounds-manager/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  fontconfig,
+  zlib,
+  icu,
+  libX11,
+  libXext,
+  libXi,
+  libXrandr,
+  libICE,
+  libSM,
+  openssl,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "muse-sounds-manager";
+  version = "1.1.0.587";
+
+  # Use web.archive.org since the upstream does not provide a stable
+  # (versioned) URL.  To see if there are new versions, visit
```

If I understand correctly how it goes, perhaps it'd be more accurate:

```suggestion
  # (versioned) URL.  To see if there are new versions, we can only rely upon
  #  the web.archive.org's has archived the new version. To check that, Visit
```

---

I have reworded this a bit to make it more clear.